pom.xml

```diff
@@ -399,6 +399,21 @@
       <artifactId>unused</artifactId>
       <version>1.0.0</version>
     </dependency>
+    <dependency>
```

Comment:
Is this the right place?

Reply:
https://github.com/apache/spark/blob/a28d2d7e900e8e2c65953fddbf32708802c4a700/pom.xml#L2691-L2695

I think we should add `exclusions` here